src/components/ContactCard/ContactForm.jsx

```diff
@@ -77,7 +77,62 @@ const initialFieldValues = fields.reduce((initialValues, { name, isArray }) => {
   return initialValues
 }, {})
 
-class ContactForm extends React.Component {
+export const moveToHead = shouldBeHead => items =>
+  items.reduce((arr, v) => (shouldBeHead(v) ? [v, ...arr] : [...arr, v]), [])
+
+const movePrimaryToHead = moveToHead(v => v.primary)
+
```

Comment:
This is something we should move outside of this file. Pretty sure it's already planned :D

Reply:
I would like to but I can't find the right place for now :smile: Maybe a new file in `helpers/` I guess

Reply:
moved to `contactToFormValues.js`